platform/security/pdp/security-pdp-authzrealm/src/main/java/ddf/security/pdp/realm/SimpleAuthzRealm.java

```diff
@@ -250,10 +240,12 @@ private boolean isPermitted(PrincipalCollection subjectPrincipal, Permission per
         }
         if (!CollectionUtils.isEmpty(perms)) {
             if (permission instanceof KeyValuePermission) {
-                permission = new KeyValueCollectionPermission((KeyValuePermission) permission);
+                permission = new KeyValueCollectionPermission(CollectionPermission.NO_ACTION,
```

Comment:
We should log a warning that the action is unknown.  We could also log a debug that warns a developer to use a KeyValueCollectionPermission instead of a KeyValuePermission directly.